src/webextension/list/manage/components/item-details.css

```diff
@@ -3,16 +3,47 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .item-details {
-  padding: 2em;
+  padding: 1em 4.6em 4.6em;
   background-color: #f9f9fa;
 }
 
 .item-details > h1 {
   font-weight: 300;
   font-size: 22px;
+  margin: 0 0 1.5em;
+  padding-bottom: 1em;
+  border-bottom: 1px solid #d7e7db;
 }
 
 .buttons {
   margin-top: 1em;
   padding: 1em 0;
 }
+
+input[name="password"] + div > span > button {
+	text-indent: -9999px;
+	padding: 0;
+	width: 32px;
+}
+
+input[name="password"] + div > span > button:before {
+	background-repeat: no-repeat;
+	background-size: 16px 16px;
+	content: "";
+	float: right;
+	height: 16px;
+	width: 16px;
+	margin-right: 8px;
+}
+
+input[name="password"] + div > span:first-child > button:before {
+	background-image: url(/icons/show.svg);
+}
+
+input[name="password"] + div > span:last-child > button:before {
+	background-image: url(/icons/hide.svg);
+}
+
+textarea {
```

Comment:
As mentioned above, we use CSS class names here instead of tag names. There isn't a class name set on the textarea element in `item-fields.js`, but you can just set className to `notes` and then update this.